docs/topics/maven.md

```diff
@@ -23,6 +23,27 @@ To use JDK 17, in your `.mvn/jvm.config` file, add:
 --add-opens=java.base/java.io=ALL-UNNAMED
 ```
 
+## Declare repositories
+
+To access artifacts in repositories that can be consumed by Maven builds, specify the ID and URL of each repository in
+the `<repositories>` tag:
+
+```xml
+<repositories>
+    <repository>
+        <id>mavenCentral</id>
+        <url>https://repo1.maven.org/maven2/</url>
```

Comment:
AFAIK Maven uses Central by default, not sure if this is a good example, especially for beginners.  
They will add the Central repository for no added value.

Reply:
Oh well, this one apparently is already done. Sorry!

Reply:
No problem! 😄